mods/ts/rules/nod-support.yaml

```diff
@@ -88,16 +88,18 @@ NAOBEL:
 		Type: Heavy
 	RevealsShroud:
 		Range: 8c0
-	RenderBuildingCharge:
-		ChargeAudio: obelpowr.aud
 	Armament:
 		Weapon: LaserFire
-		LocalOffset: 0,0,725
+		LocalOffset: 1500,250,1250
```

Comment:
This is a bit too high and too the right:

![screen shot 2015-05-02 at 12 03 21 2](https://cloud.githubusercontent.com/assets/167819/7439047/5d814c26-f0c3-11e4-94ba-cb93ed0997fc.png)